okta/provider.go

```diff
@@ -81,39 +81,45 @@ func Provider() terraform.ResourceProvider {
 				Type:        schema.TypeString,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("OKTA_BASE_URL", "okta.com"),
-				Description: "The Okta url. (Use 'oktapreview.com' for Okta testing)",
+				Description: "The Okta url. (Use 'oktapreview.com' for Okta testing).",
 			},
 			"backoff": {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     true,
 				Description: "Use exponential back off strategy for rate limits.",
 			},
-			"min_wait_seconds": {
-				Type:        schema.TypeInt,
-				Optional:    true,
-				Default:     30,
-				Description: "minimum seconds to wait when rate limit is hit. We use exponential backoffs when backoff is enabled.",
-			},
 			"max_wait_seconds": {
 				Type:        schema.TypeInt,
 				Optional:    true,
 				Default:     300,
-				Description: "maximum seconds to wait when rate limit is hit. We use exponential backoffs when backoff is enabled.",
+				Description: "Maximum seconds to wait when rate limit is hit. We use exponential backoffs when backoff is enabled.",
```

Comment:
@bogdanprodan-okta Is this description correct? It seems to me that it sets just timeout for SDK client...

Reply:
yeah, the description is correct, but the value is misused. It should be:
```okta.WithRateLimitMaxBackOff(int64(c.maxWait)),```
instead of
```okta.WithRequestTimeout(int64(c.maxWait))```
As for request timeout, it should be either `0` or there should be an additional config introduced.

Reply:
Thanks for the very fast answer! I introduced `request_timeout` and fixed `max_wait_seconds` properties